bindings/python/test_m68k.py

```diff
@@ -40,12 +40,14 @@
 	16: "Absolute Data Addressing  - Short",
 	17: "Absolute Data Addressing  - Long",
 	18: "Immidate value",
-}    
+}
 
 def print_insn_detail(insn):
     if len(insn.operands) > 0:
         print("\top_count: %u" % (len(insn.operands)))
 
+	print("\tgroups_count: %u" % len(insn.groups))
```

Comment:
Hum... the groups_counts work when I test this in python and prints the correct value.